java/src/main/java/org/rocksdb/LRUCache.java

```diff
@@ -69,14 +69,14 @@ public LRUCache(final long capacity, final int numShardBits,
    * @param highPriPoolRatio percentage of the cache reserves for high priority
    *     entries
    */
-  public LRUCache(final long capacity, final int numShardBits,
-      final boolean strictCapacityLimit, final double highPriPoolRatio) {
-    super(newLRUCache(capacity, numShardBits, strictCapacityLimit,
-        highPriPoolRatio));
+  public LRUCache(final long capacity, final int numShardBits, final boolean strictCapacityLimit,
+      final double highPriPoolRatio, final double lowPriPoolRatio) {
+    super(newLRUCache(
+        capacity, numShardBits, strictCapacityLimit, highPriPoolRatio, lowPriPoolRatio));
```

Comment:
I think the idea here is to make it possible to construct an `LRUCache` with 1, 2, 3, 4, or all 5 of the parameters. So we would want to keep the version that takes `capacity`...`highPriPoolRatio` and add a new one that takes all 5

Reply:
Done, Thanks.